### DISCUSSION TITLE
Custom named colors for each theme.

### DISCUSSION THREAD

**Discussion by RealRONiN** [saadeghi/daisyui#2565] [Q&A] [answered]
Hello, how can I add custom named colors for each diff theme I have?

Example:

```js
  daisyui: {
    themes: [
      {
        light: {
          ...require("daisyui/src/theming/themes")["light"],
          "search-lg": "#f4f5f6"
        },
        dark: {
          ...require("daisyui/src/theming/themes")["dark"],
          "search-lg": "#2b2e33"
        },
      },
    ],
  },
```

**Comment by RealRONiN** [comment]
Hey @saadeghi sorry for the mention, but this went unanswered and I can't find Daisy's Discord server either.

**Reply by saadeghi** [reply, marked as answer]
No problem.  
It's in FAQ: https://github.com/saadeghi/daisyui/discussions/1949

And here's the tutorial:  
https://daisyui.com/blog/how-to-add-new-colors-to-daisyui/

**Reply by RealRONiN** [reply]
That's perfect TYSM!
